internal/distro/fedora31/distro_test.go

```diff
@@ -137,3 +140,187 @@ func TestImageType_BuildPackages(t *testing.T) {
 		}
 	}
 }
+
+func TestImageType_Size(t *testing.T) {
+	const gigaByte = 1024 * 1024 * 1024
+	sizeMap := []struct {
+		name       string
+		inputSize  uint64
+		outputSize uint64
+	}{
+		{
+			name:       "ami",
+			inputSize:  6*gigaByte + 1,
+			outputSize: 6*gigaByte + 1,
+		},
+		{
+			name:       "ami",
+			inputSize:  0,
+			outputSize: 6 * gigaByte,
+		},
+		{
+			name:       "vhd",
+			inputSize:  10 * gigaByte,
+			outputSize: 10 * gigaByte,
+		},
+		{
+			name:       "vhd",
+			inputSize:  10*gigaByte - 1,
+			outputSize: 10 * gigaByte,
+		},
+	}
+
+	distro := fedora31.New()
+	arch, err := distro.GetArch("x86_64")
+	assert.Nil(t, err)
+	if err == nil {
```

Comment:
that would be `if assert.NoError(t, err)`

Reply:
Personally, I would prefer using `require.NoError(t, err)` and dropping the `if`.

Reply:
Thanks for the suggestion to use the assertion within the condition statement @atodorov. @ondrejbudai although it wouldn't be a very big difference, using `require` instead of `assert` wouldn't be functionally equivalent, as any failure would mean the whole test fails, as opposed to the current situation where the test can continue with another test scenario within the same test.

I'll fix the asserts and will also create a PR to backport the same changes to the already merged PR for F30.

Reply:
Using 
```
arch, err := distro.GetArch("x86_64")
require.NoError(err)
```

is fine in `TestImageType_Size`, because if `err != nil`, the rest of the test is skipped.

The same in `TestImageType_BasePackages`: if `distro.GetArch` fails, there's no point in continuing the test because the `arch` variable is probably `nil`.